### PR TITLE
WIP:feature/accordion

### PR DESCRIPTION
**Pull request by ghost**
intial

### CODE REVIEW

**Comment by ghost** [comment]
initial commit - check to see if working.

File: src/components/Accordion/Accordion.stories.tsx [hunk 1]
```diff
@@ -1,29 +1,91 @@
...
+            expandIcon={<Icon name='angle down' size='mini' color='black' />}
+            disabled={true}
+        />
+    </div>
+);
+
+export const ControlledAccordion = () => {
+
+    const [expanded, setExpanded] = useState(false);
+    // const handleChange =
```
Comment (kate620) [inline src/components/Accordion/Accordion.stories.tsx]: For production maybe better to take out the comments.

File: src/components/Accordion/Accordion.stories.tsx [hunk 2]
```diff
@@ -1,29 +1,91 @@
...
+      expanded={true}
+    />
+    <Accordion
+      title="Accordion 3"
+      content="This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks"
```
Comment (kate620) [inline src/components/Accordion/Accordion.stories.tsx]: Maybe a good way to pull content from another component so it won't be too long.

File: src/components/Accordion/Accordion.stories.tsx [hunk 3]
```diff
@@ -1,29 +1,91 @@
...
+        <Accordion
+            className="accordion"
+            title="Accordion title"
+            expandIcon={<Icon name='angle down' size='mini' color='black' />}
+            disabled={true}
+        />
+    </div>
+);
+
+export const ControlledAccordion = () => {
```
Comment (kate620) [inline src/components/Accordion/Accordion.stories.tsx:39]: It doesn't work.

File: src/components/Accordion/Accordion.stories.tsx [hunk 4]
```diff
@@ -1,29 +1,91 @@
...
+    // (panel: string) => (e: React.SyntheticEvent, isExpanded: boolean) => {
+    //     setExpanded(isExpanded ? panel : false);
+    //   };
+
+      
+    return (
+    <div>
+        <p>A basic Accordion component.</p>
+        <Accordion
+            onChange={() => setExpanded(!expanded)}
```
Comment (kate620) [inline src/components/Accordion/Accordion.stories.tsx]: What does onChange do?

File: src/components/Accordion/Accordion.tsx [hunk 5]
```diff
@@ -1,95 +1,90 @@
...
+    disabled,
+    expanded,
+    controlledExpanded,
+    onChange,
+    ...rest
+  } = props;
+
+  const [expand, setExpand] = useState(expanded);
+
+  const closeAll = () => {
```
Comment (kate620) [inline src/components/Accordion/Accordion.tsx]: Bloat code

**Review by kate620 — COMMENTED** [review]
Just make some changes you will be good to go.

**Pull request closed without merging**